cli/src/subcommand.rs

```diff
@@ -167,5 +174,10 @@ pub fn check_and_comment_on_pr(
             &comment_body,
         )?;
     }
+
+    if !is_empty && exit_on_error {
```

Comment:
isn't `!is_empty` always true because we'll return on line 143 if `is_empty` is true?
```suggestion
    if exit_on_error {
```

Reply:
You're absolutely right! Done in e5de367041d437a4219f901dab8cfc1b4e726727.